### PR TITLE
Main working

### PR DESCRIPTION
**Pull request by I4LYT**

### CODE REVIEW

**Review by Extraskilled56 — CHANGES_REQUESTED** [review]
Change The "<title>My Website</title>" to "<title>Games</title" please so it fits the site

File: games/index.html
Comment (Extraskilled56) [inline games/index.html:1]: this is the file that needs fixing

**Comment by I4LYT** [comment]
OK

**Comment by I4LYT** [comment]
done

**Review by Extraskilled56 — APPROVED** [review]
thanks

**Pull request merged**